jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -404,6 +404,9 @@ void java_bytecode_convert_method_lazy(
   }
 
   method_symbol.type=member_type;
+  // Not used in jbmc at present, but other codebases that use jbmc as a library
+  // use this information.
+  method_symbol.type.set(ID_C_abstract, m.is_abstract);
```

Comment:
Why do you introduce a new irep_id? There is already `ID_abstract`, which is used for marking abstract classes: https://github.com/diffblue/cbmc/blob/72e99a0d162a152362ec84cd41c4e6c1be114adf/src/util/std_types.h#L415